lib/publiccloud/instance.pm

```diff
@@ -350,7 +350,7 @@ sub wait_for_guestregister {
 
 =head2 wait_for_ssh
 
-    wait_for_ssh([timeout => 600] [, proceed_on_failure => 0] [, ...])
+    wait_for_ssh([timeout => 600] [, proceed_on_failure => 0] [, scan_ssh_host_key => 0] [, ...])
```

Comment:
is it possible to have this PR splitted in 2? Is it possible to have splitted like:
- ssh configg for SAP
- scan_ssh_host_key

Reply:
This is unfortunately too connected and quite expensive to do verification runs for.

Can we please keep this together then?